pkg/scalers/ibmmq_scaler.go

```diff
@@ -34,14 +34,15 @@ type IBMMQScaler struct {
 
 // IBMMQMetadata Metadata used by KEDA to query IBM MQ queue depth and scale
 type IBMMQMetadata struct {
-	host             string
-	queueManager     string
-	queueName        string
-	username         string
-	password         string
-	targetQueueDepth int64
-	tlsDisabled      bool
-	scalerIndex      int
+	host                       string
+	queueManager               string
+	queueName                  string
+	username                   string
+	password                   string
+	targetQueueDepth           int64
+	activationTargetQueueDepth int64
```

Comment:
```suggestion
	queueDepth           int64
	activationQueueDepth int64
```
Wouldn't it be better to rename these fields, so the reflect the real vaule in the spec?

Reply:
you are right! I did the change as a robot without thinking anything more